mautic.gemspec

```diff
@@ -14,23 +14,24 @@ Gem::Specification.new do |s|
   s.description = 'Rails client for Mautic API. Provide wrapper for push to mautic form'
   s.license     = 'MIT'
 
-  s.required_ruby_version = '>= 2.3'
+  s.required_ruby_version = '>= 2.7'
```

Comment:
pls 2.5 pokud s tím není zásadní problém, je to minimum pro rails 6 a stále podporujeme